client/account-settings/view.js

```diff
@@ -37,9 +38,9 @@ const AccountSettingsRootView = ( { formData, formMeta, actions, noticeActions,
 	const renderContent = () => {
 		if ( ! formData && ! formMeta.isFetching ) {
 			return (
-				<p className="error-message">
+				<ErrorMessage>
```

Comment:
We only have 2 `<ErrorMessage />`s - maybe it's not worth creating a new component for it?